ctapipe/reco/tests/test_shower_processor.py

```diff
@@ -1,18 +1,28 @@
 """
 Tests for ShowerProcessor functionalities.
 """
-from numpy import isfinite
-import pytest
+from copy import deepcopy
 
+import pytest
+from numpy import isfinite
 from traitlets.config.loader import Config
 
 from ctapipe.calib import CameraCalibrator
 from ctapipe.image import ImageProcessor
 from ctapipe.reco import ShowerProcessor
 
 
-@pytest.mark.parametrize("reconstructor_type", ["HillasReconstructor", "HillasIntersection"])
-def test_shower_processor_geometry(example_event, example_subarray, reconstructor_type):
+@pytest.mark.parametrize(
+    "reconstructor_types",
+    [
+        ["HillasReconstructor"],
+        ["HillasIntersection"],
+        ["HillasReconstructor", "HillasIntersection"],
```

Comment:
To be future-proof here, you might want to use:
```py3
list(Reconstructor.non_abstract_subclasses().keys())
```
rather than an explicit list.  That way if anyone adds a new reconstructor, it is tested